Source/WebKit/WebProcess/WebPage/FindController.cpp

```diff
@@ -78,7 +78,8 @@ FindController::~FindController()
 
 PluginView* FindController::mainFramePlugIn()
 {
-    return m_webPage->mainFramePlugIn();
+    RefPtr protectedWebPage { m_webPage.get() };
+    return protectedWebPage->mainFramePlugIn();
```

Comment:
It seems there an are a few places in this PR where we could avoid extra variables by introducing a new `protectedWebPage()` member function so I think we should do that. Local variables are normally used to avoid repeated calls to `protectedFoo()` functions within the same function.